assets/css/structures/_grid.scss

```diff
@@ -64,5 +64,10 @@
 
   a {
     color: white;
+
+    &:focus {
+      background-color: pink;
```

Comment:
Could we add this a sass variable with its intended purpose (i.e.,`$lt-background-color`; however you want to develop the naming system is fine with me). so that eventually when we activate dark mode we can easily switch these things globally as a theme? Thank you!